workflow/rules/common.smk

```diff
@@ -459,16 +421,26 @@ def get_reference_name(reference_file):
 
 if not VPIPE_BENCH:
     reference_file = config["input"]["reference"]
-    if not os.path.isfile(reference_file):
+    if not reference_file:
+        raise ValueError(
+            f"ERROR: No input reference in configuration. Please read: config/README.md or https://github.com/cbg-ethz/V-pipe/tree/master/config"
```

Comment:
`config/README.md` does not exist in this PR, right?

Reply:
Yes... I'll move it to feature-documentation. :sweat_smile: